src/DIRAC/MonitoringSystem/Client/DataOperationSender.py

```diff
@@ -0,0 +1,82 @@
+import DIRAC
+from DIRAC import S_OK, S_ERROR, gLogger
+
+from DIRAC.ConfigurationSystem.Client.Helpers.Operations import Operations
+from DIRAC.AccountingSystem.Client.DataStoreClient import gDataStoreClient
+from DIRAC.AccountingSystem.Client.Types.DataOperation import DataOperation
+from DIRAC.MonitoringSystem.Client.MonitoringReporter import MonitoringReporter
+
+
```

Comment:
Instead of using directly `gLogger` below, do:

```suggestion
sLog = gLogger.getSubLogger(__name__)
```